classes/publication/enums/JavStage.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/**
+ * @file publication/enums/JavStage.php
```

Comment:
Because this refers to an external standard, it would be good to add a URL to where it's specified.